src/main/java/com/contrastsecurity/sdk/ContrastSDK.java

```diff
@@ -66,16 +67,21 @@
     private UrlBuilder urlBuilder;
     private Gson gson;
     Proxy proxy;
-    
+    private String integrationName;
+    private String version;
+
     private int connectionTimeout = DEFAULT_CONNECTION_TIMEOUT;
     private int readTimeout = DEFAULT_READ_TIMEOUT;
 
+    @Getter

```

Comment:
This is probably fine for pragmatism, but when we are using the sdk, we should try to avoid treating the builder like a repository/holder of values.